packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -151,6 +151,78 @@ export interface CronPayload {
   cron_org: string;
 }
 
+export interface BotSecrets {
+  privateKey: string;
+  appId: string;
+  secret: string;
+}
+
+/**
+ * A helper for fetch secret from SecretManager.
+ */
+async function getBotSecrets(): Promise<BotSecrets> {
+  const projectId = process.env.PROJECT_ID;
+  const functionName = process.env.GCF_SHORT_FUNCTION_NAME;
+  const secretsClient = new SecretManagerV1.SecretManagerServiceClient();
+  const [version] = await secretsClient.accessSecretVersion({
+    name: `projects/${projectId}/secrets/${functionName}/versions/latest`,
+  });
+  // Extract the payload as a string.
+  const payload = version?.payload?.data?.toString() || '';
+  if (payload === '') {
+    throw Error('did not retrieve a payload from SecretManager.');
+  }
+  const secrets = JSON.parse(payload);
+
+  if (Object.prototype.hasOwnProperty.call(secrets, 'cert')) {
+    secrets.privateKey = secrets.cert;
+    delete secrets.cert;
+  }
+  if (Object.prototype.hasOwnProperty.call(secrets, 'id')) {
+    secrets.appId = secrets.id;
+    delete secrets.id;
+  }
+  return {
+    privateKey: secrets.privateKey,
+    appId: secrets.appId,
+    secret: secrets.secret,
```

Comment:
Along with my requested renaming above, we could check for `secrets.webhookSecret ?? secrets.secret` for backwards and forwards compatibility (allow putting the the value in either `secret` or `webhookSecret` in the stored JSON contents).

Reply:
Done